### PR TITLE
feat(dashboard-refresh): add DashboardRefreshDropdown and DashboardRefresher

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
- DashboardRefreshDropdown
    image

- DashboardRefresher
    image

I only worked on markup for translation.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "@amcharts/amcharts5": "^5.2.31",
     "@amcharts/amcharts5-geodata": "^5.0.6",
     "@gtm-support/vue2-gtm": "^1.3.0",
-    "@spaceone/design-system": "^1.1.0-beta.462",
+    "@spaceone/design-system": "^1.1.0-beta.464",
```
Comment (minseolee) [inline package.json:30]: Thanks!

File: src/services/dashboards/modules/DashboardRefreshDropdown.vue [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    }
+
+    /* custom design-system component - p-select-dropdown */
+    :deep(.currency-select-dropdown) {
+        .dropdown-button {
+            border-top-left-radius: 0;
+            border-bottom-left-radius: 0;
+        }
+    }
+}
```
Comment (WANZARGEN) [inline src/services/dashboards/modules/DashboardRefreshDropdown.vue:126]: think: T.T I know there was no choice...

File: src/services/dashboards/modules/DashboardRefreshDropdown.vue [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+                       :animation="loading ? 'reserve-spin' : undefined"
+        />
+        <p-select-dropdown class="currency-select-dropdown"
+                           :items="intervalItems"
+                           :selected="interval"
+                           :read-only="loading"
+                           :class="{ loading }"
+                           @select="handleSelectInterval"
+        >
+            <span v-if="!filled">d</span>
```
Comment (minseolee) [inline src/services/dashboards/modules/DashboardRefreshDropdown.vue:16]: What is this `d` ?

**Review by minseolee — APPROVED** [review]
LGTM!

**Review by WANZARGEN — APPROVED** [review]
Nice clean code :)

File: src/services/dashboards/modules/DashboardRefreshDropdown.vue (hunk 2 above)
Comment (sulmoJ) [reply src/services/dashboards/modules/DashboardRefreshDropdown.vue:126]: It was my best 😭

**Pull request merged**